apps/paper/src/LottieAnimatedExample.tsx

```diff
@@ -58,6 +63,10 @@ const LottieAnimatedExample = () => {
   const onToggleImperative = () => setImperative(p => !p);
 
   const startImperative = () => {
+    if (Platform.OS === 'web') {
+      console.warn('Imperative API is not supported on web');
+      return;
+    }
```

Comment:
Does it make sense to move this piece of code to the library code? This way user on web knows that it's not supported instead of a silent no-op.